spec/views/rb_taskboards/show_spec.rb

```diff
@@ -101,13 +110,12 @@
   end
 
   describe 'story blocks' do
-
     it 'contains the story id' do
       render
 
       stories.each do |story|
         expect(rendered).to have_selector "#story_#{story.id}" do
-          with_selector ".id", Regexp.new(story.id.to_s)
+          with_selector '.id', Regexp.new(story.id.to_s)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.